content/v2.0/router/overview.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Router: Overview"
+order: 10
+---
+
+# Overview
+
+Hanami applications use [Hanami::Router](https://github.com/hanami/router) for routing: a Rack compatible, lightweight and fast HTTP router for Ruby.
+
+## The first route
+
+With your favorite editor open `/config/routes.rb` and add the following line inside of the `main` slice block.
+
+```ruby
+get '/hello', to: ->(env) { [200, {}, ['Hello from Hanami!']] }
+```
+
+So the whole file would look like this:
+
+```ruby
+# /config/routes.rb
+
+Hanami.application.routes do
+  slice :main, at: "/" do
+    root to: "home.show"
+
+    get '/hello', to: ->(env) { [200, {}, ['Hello from Hanami']] }
+  end
+end
+```
+
+Then start the server with `puma config.ru` and visit [http://localhost:3000/hello](http://localhost:2300/hello).
```

Comment:
```suggestion
Then start the server with `bundle exec rackup` and visit [http://localhost:3000/hello](http://localhost:2300/hello).
```

I'm not sure but I think this will do the same thing when you have puma configured, and run whatever other server you have configured if you don't.

Reply:
I believe this will be changed to sth like `hanami start` relatively soon. The CLI is not yet done, but I don't expect these guides to be finished earlier. @cllns what do you think?

Reply:
Yea that makes sense. Can we come up with some kind of tag to make sure we go through and update these? `<!-- FIXME: Change to `bundle exec hanami server` when it's ready -->`
Though in this case, we know exactly what the command will be, so maybe we should just write it in here, even though it doesn't work yet.